package.json

```diff
@@ -3,8 +3,9 @@
   "version": "0.0.1",
   "description": "Cracking the Coding Interview - Answers in Javascript",
   "scripts": {},
-  "dependencies": {},
+  "dependencies": {
+    "eslint": "^3.14.1"
```

Comment:
`eslint` is only used for development (testing). As such, it would fit better in the `devDependencies`